tests/testthat/test-sparsevctrs.R

```diff
@@ -0,0 +1,60 @@
+test_that("sparse tibble can be passed to `fit() - recipe", {
+  skip_if_not_installed("xgboost")
+
+  hotel_data <- sparse_hotel_rates()
+  hotel_data <- sparsevctrs::coerce_to_sparse_tibble(hotel_data)
+
+  spec <- parsnip::boost_tree() %>%
+    parsnip::set_mode("regression") %>%
+    parsnip::set_engine("xgboost")
+
+  rec <- recipes::recipe(avg_price_per_room ~ ., data = hotel_data)
+
+  wf_spec <- workflow() %>%
+    add_recipe(rec) %>%
+    add_model(spec)
+  
+  expect_no_error(
+    wf_fit <- fit(wf_spec, hotel_data)
+  )
```

Comment:
```suggestion
  expect_no_condition(wf_fit <- fit(wf_spec, hotel_data))
```

..and the same for the analogous lines below!